source/includes/_enrichment.md

```diff
@@ -1,5 +1,779 @@
+            "publicly_traded_exchange":null,
+            "logo_url":"https://zenprospect-production.s3.amazonaws.com/uploads/pictures/62299e01b86a98000152b0c5/picture",
+            "crunchbase_url":null,
+            "primary_domain":"apollo.io",
+            "persona_counts":{
+               
+            },
+            "domain":"apollo.io",
+            "team_id":"6181a50999668600ded6fe71",
+            "organization_id":"5e66b6381e05b4008c8331b8",
+            "account_stage_id":"6181a50999668600ded6fe7c",
+            "source":"deployment",
+            "original_source":"deployment",
+            "owner_id":"6181a50a99668600ded6feed",
+            "created_at":"2021-11-03T16:18:47.229Z",
+            "phone_status":"no_status",
+            "test_predictive_score":null,
+            "hubspot_id":null,
+            "salesforce_id":null,
+            "crm_owner_id":null,
+            "parent_account_id":null,
+            "sanitized_phone":"+12023741312",
+            "account_playbook_statuses":[
+               {
+                  "_id":"61844b47a4258e00c2499c8e",
+                  "added_by_id":"6181a50a99668600ded6feed",
+                  "added_on":"2021-11-04T21:06:15.053+00:00",
+                  "completed_step_ids":[
+                     "6184391f26e0aa00a4ccc56b"
+                  ],
+                  "created_at":null,
+                  "current_step_id":"61844dba8b94530112288546",
+                  "finished_reason_cd":"completed_all_steps",
+                  "paused_reason_cd":null,
+                  "playbook_id":"6184388a0515e4008cbf5129",
+                  "position":2,
+                  "status_cd":"finished",
+                  "updated_at":null,
+                  "id":"61844b47a4258e00c2499c8e",
+                  "key":"61844b47a4258e00c2499c8e"
+               }
+            ],
+            "existence_level":"full",
+            "label_ids":[
+               
+            ],
+            "typed_custom_fields":{
+               "618438b906b01300da086546":"Google, Okta, Lyft"
+            },
+            "modality":"account"
+         },
+         "phone_numbers":[
+            {
+               "raw_number":"+1 415-763-6055",
+               "sanitized_number":"+14157636055",
+               "type":"work_hq",
+               "position":0,
+               "status":"no_status"
+            }
+         ]
+      }
+   ]
+}
+```
+
+This endpoint enriches people information in bulk - the more information you pass in, the more likely we can find a match. 
+
+### Credit Usage
+
+The enrich endpoint charges you credits for its usage. If a verified email is successfully returned, it will cost you 1 credit. If an email is not found, but Apollo successfully found ALL of the following information: Name, Linkedin Profile, Current Company Information, Apollo will charge a fraction of a credit. Typically this is 0.01 credit per successful enrichment without email. But it may be higher depending on your specific plan.
```

Comment:
Please be explicit in below things

- The limits for this api are 1/10 th of original enrich api.
- The number records that can be enriched in single go is only 10
- Duplicates will not be charged within the request